src/main/java/seedu/address/logic/commands/EditFolderNameCommand.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.folder.Folder;
+
+/**
+ * Edits the name of an existing folder in the address book.
+ */
+public class EditFolderNameCommand extends Command {
+
+    public static final String COMMAND_WORD = "mv";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Renames the folder identified by the old "
+            + "folder name used in the displayed folders list.\n"
+            + "Parameters: OLD_FOLDER_NAME (Name must be an existing folder)"
+            + " + NEW_FOLDER_NAME (Name must not already exist in folder list)\n"
+            + "Example: " + COMMAND_WORD + " OLD_FOLDER_NAME | " + "NEW_FOLDER_NAME";
+
+    public static final String MESSAGE_SUCCESS_EDIT_FOLDER_NAME = "Folder updated to: %1$s";
+    public static final String MESSAGE_NO_SUCH_FOLDER = "No such folder found in UNIon to be renamed";
+    public static final String MESSAGE_FOLDER_ALREADY_EXISTS = "Cannot rename folder to"
+            + " an already existing folder in UNIon";
+
+    private final Folder folderToRename;
+    private final Folder folderToReplaceOld;
```

Comment:
Is it possible to rename the local variables:
`folderToRename` --> `oldFolder`
`folderToReplaceOld` --> `newFolder`

Then this will also be consistent with your `setNewFolder` method which has parameter names same as the above.

Reply:
okie noted